workspace_functions.go

```diff
@@ -0,0 +1,23 @@
+package databricks
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/databricks/databricks-sdk-go/httpclient"
+)
+
+// CurrentWorkspaceId returns the workspace ID of the workspace that this client is
+// connected to.
+func (w *WorkspaceClient) CurrentWorkspaceId(ctx context.Context) (int64, error) {
+	var workspaceIdStr string
+	err := w.apiClient.Client.Do(ctx, "GET", "/api/2.0/preview/scim/v2/Me", httpclient.WithResponseHeader("X-Databricks-Org-Id", &workspaceIdStr))
```

Comment:
Is this a stable interface?

Reply:
I'm raising internally, will send you the links where I'm asking offline.